js/components/posts_index.react.js

```diff
@@ -1,9 +1,17 @@
 var React = require('react');
+var Fixtures = require('../fixtures/post-data');
 
 var PostsIndex = React.createClass({
   render: function() {
     return (
+    <div>
       <h1>hi i am a posts index!</h1>
+      <ul>
+        {Fixtures.map(function(item) {
+        return <li>{item}</li>
```

Comment:
+ indent line 11
+ make each link an `<a>` element with the item's title as text and the item's URL as href